pinot-query-planner/src/main/java/org/apache/pinot/query/planner/plannode/SetOpNode.java

```diff
@@ -57,7 +57,7 @@ public boolean isAll() {
 
   @Override
   public String explain() {
-    return _setOpType.toString();
+    return _all ? _setOpType.toString() + "_ALL" : _setOpType.toString();
```

Comment:
Does it works in `Union`? AFAIR union always behaves as `all = true`. Is this attribute always true in `union`?